docs/source/blocks/block-style-wrapper.md

```diff
@@ -165,6 +167,54 @@ data like:
 
 will generate classnames `primary inverted`
 
+## Injecting custom CSS properties
+
+The style wrapper also allows to inject custom CSS properties, using the converter syntax.
+This use case is useful in some scenarios where the property that you are injecting is generic, customizable per project.
+
+```
+{
+  "styles": {
+    "backgroundColor:CSSProperty": "#222",
```

Comment:
Could this be written as 
```suggestion
    "--background-color": "#222",
```
instead? Neither one can be accessed without square brackets in javascript, so I don't see a benefit to transforming the name to something else. Writing it the same way as the actual CSS property seems clearer to me.

Reply:
The converters detection system now works with sufixes, colon separated. Are you saying that you'd prefer to add another detection system based on if '--' is present?

Reply:
It's a good point though...

Reply:
@davisagli done, it was easy than expected.